fix: fix and update the tutorials for `0.17` (#744)
cosmos/sdk-tutorials: launchpad-to-stargate/tutorial/01-introduction.md

Start off by scaffolding your application using the following command.

```sh
starport scaffold chain github.com/user/pofe
```

> Once this is done, you should have a folder called `pofe` with your boilerplate Stargate application.

Reviewer: ```suggestion
After the scaffold command is run, you should have a folder called `pofe` with your boilerplate Stargate application.
```

Reviewer: - let's use "after" instead of "once" 
- let's specify the subject "the scaffold command" instead of using "it"
